functions-python/validation_report_processor/src/main.py

```diff
@@ -128,10 +128,10 @@ def generate_report_entities(
     logging.info(f"Creating validation report entities for {report_id}.")
 
     html_report_url = (
-        f"{FILES_ENDPOINT}/{feed_stable_id}/{dataset_stable_id}/report.html"
+        f"{FILES_ENDPOINT}/{feed_stable_id}/{dataset_stable_id}/report-{version}.html"
     )
     json_report_url = (
-        f"{FILES_ENDPOINT}/{feed_stable_id}/{dataset_stable_id}/report.json"
+        f"{FILES_ENDPOINT}/{feed_stable_id}/{dataset_stable_id}/report-{version}.json"
```

Comment:
[picky-suggestion]: replace `.`s with `_`s in the version

Reply:
done ✅